src/api/db/models/Camera.ts

```diff
@@ -104,44 +123,45 @@ export class CameraModel {
         await cam.save();
         successfulOps.push({ op: 'cam-registered', info: { cameraId: input.cameraId } });
         const wirelessCameras = await CameraModel.getWirelessCameras({}, context);
-        const project = await ProjectModel.createCameraConfig({
-          projectId,
-          cameraId: cam._id
-        }, context);
+        const project = await ProjectModel.createCameraConfig(
+          {
+            projectId,
+            cameraId: cam._id,
+          },
+          context,
+        );
 
         return { wirelessCameras, project };
-
-      }
-      else {
-        const msg = activeReg.projectId === projectId
-          ? `This camera is already registered to the ${projectId} project!`
-          : 'This camera is registered to a different project!';
+      } else {
+        const msg =
+          activeReg?.projectId === projectId
+            ? `This camera is already registered to the ${projectId} project!`
+            : 'This camera is registered to a different project!';
         throw new CameraRegistrationError(msg, {
-          currProjReg: activeReg.projectId
+          currProjReg: activeReg?.projectId,
         });
       }
-
     } catch (err) {
       // reverse successful operations
       for (const op of successfulOps) {
         if (op.op === 'cam-registered') {
-          await CameraModel.unregisterCamera(
-            { cameraId: op.info.cameraId }, context
-          );
+          await CameraModel.unregisterCamera({ cameraId: op.info.cameraId }, context);
         }
       }
 
       if (err instanceof GraphQLError) throw err;
-      throw new InternalServerError(err);
+      throw new InternalServerError(err as string);
     }
   }
 
-  static async unregisterCamera(input, context) {
-    const successfulOps = [];
+  static async unregisterCamera(
+    input: { cameraId: string },
+    context: Context,
```

Comment:
Not that it matters a ton,  but this one is autogenerated and exported from `src/@types/graphql.ts` as `UnregisterCameraInput`.